### PR TITLE
Add HDD Id

### PR DESCRIPTION
**Pull request by GewoonJaap**

### CODE REVIEW

**Comment by GewoonJaap** [comment]
Added the HDD/SSD Id and added the application/json header so browsers properly render the JSON data.
It is now easier to add ini data, since its a unified function

**Comment by NebN** [comment]
Thanks for the PR, I'll take a look as soon as I am able.

File: cmd/main.go [hunk 1]
```diff
@@ -130,9 +130,11 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			slog.String("error", err.Error()),
 			slog.String("attempting to marshal", fmt.Sprintf("%+v\n", response)))
 		errorResponse, _ := json.Marshal(newErrorReport(err.Error()))
+		w.Header().Set("Content-Type", "application/json")
 		w.Write([]byte(errorResponse))
 	} else {
 		slog.Debug("Responding to request", "response", responseJson)
+		w.Header().Set("Content-Type", "application/json")
```
Comment (NebN) [inline cmd/main.go:137]: `w.Header().Set("Content-Type", "application/json")`

could be called just once, before checking for errors, as the response is always a JSON.

File: internal/monitor/diskmonitor.go [hunk 2]
```diff
@@ -254,50 +265,63 @@ func (monitor *DiskMonitor) readZfsDatasets() map[string]ZfsDataset {
...
+            slog.Debug("Disk temp unavailable", "disk", sectionName)
+        }
+
+		spunDown, err := section.GetKey("spundown")
+
+        diskIniMap[sectionName] = DiskIni{
+            ID:   idString.String(),
+            Temp: temp,
+			Spundown: spunDown.String() == "1",
+        }
```
Comment (NebN) [inline internal/monitor/diskmonitor.go:321]: weird indentation here

File: internal/monitor/diskmonitor.go [hunk 3]
```diff
@@ -28,11 +28,21 @@ type DiskStatus struct {
...
+	Name 		string `json:"name"`
+	Temp 		uint64 `json:"temp"`
+	Id   		string  `json:"disk_id"`
+	IsSpinning  bool    `json:"is_spinning"`
+}
+
+type DiskIni struct {
+    ID   string
+    Temp uint64
+	Spundown bool
```
[the remark this answers is not in the document]
Comment (GewoonJaap) [reply internal/monitor/diskmonitor.go:45]: Yeah I guess because I just tabs. Will change to spaces

File: internal/monitor/diskmonitor.go (hunk 2 above)
Comment (GewoonJaap) [reply internal/monitor/diskmonitor.go:321]: Yeah I guess because I just tabs. Will change to spaces

File: cmd/main.go (hunk 1 above)
Comment (GewoonJaap) [reply cmd/main.go:137]: Updated

**Comment by GewoonJaap** [comment]
@NebN Resolved your comments ;)

**Review by NebN — APPROVED** [review]
Tested on my unraid setup and works as expected, thanks!

**Pull request merged**